openedx/core/djangoapps/waffle_utils/tests/test_init.py

```diff
@@ -83,20 +83,18 @@ def test_course_waffle_flag(self, waffle_enabled, course_override, result):
         (False, WaffleFlagCourseOverrideModel.ALL_CHOICES.unset, False),
     )
     @ddt.unpack
-    def test_legacy_course_waffle_flag(self, waffle_enabled, course_override, result):
+    def test_future_course_waffle_flag(self, waffle_enabled, course_override, result):
         """
-        Tests various combinations of a legacy flag being set in waffle and overridden for a course.
+        Tests various combinations of a __future__ flag being set in waffle and overridden for a course.
         """
-        test_legacy_course_flag = LegacyCourseWaffleFlag(
-            self.NAMESPACE_NAME,
-            self.FLAG_NAME,
-            __name__,
+        test_furure_course_flag = FutureCourseWaffleFlag(
```

Comment:
Minor typo in `test_furure_course_flag`, but since we're going to remove this so soon, I plan to merge as-is.

Reply:
Sorry @dyudyunov. I didn't get to this on time, and now there is a conflict. Can you please update? Feel free to fix or leave this typo. Also, let me know where the conflict was in a comment so I can double-check the fix. You can keep it squashed so I can merge when you are done. Thanks!